doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -16285,9 +16289,143 @@ SetpointManager:SingleZone:OneStageHeating,
+
+#### Field: Name
+
+This alpha field is a unique, user-assigned name for an instance of a hot water return temperature reset setpoint manager.
+
+#### Plant Loop Supply Outlet Node
+
+This alpha field is the name of the plant loop supply side outlet node for the plant being managed by this setpoint manager.  The current temperature of this node is sensed in order to calculate current loop demand.  This node is typically also where the plant loop setpoint is specified, but not necessarily.  This setpoint manager looks up the actual setpoint node from the PlantLoop internal data structures as it is specified on the PlantLoop object.  If the plant loop setpoint is on a different node than the supply outlet node, the loop may not maintain return temperature control as expected.
+
+#### Plant Loop Supply Inlet Node
 
+This alpha field is the name of the plant loop supply side inlet node whose temperature is controlled using this setpoint manager by resetting the supply setpoint temperature.  The current temperature of this node is also sensed in order to calculate current loop demand.
 
+#### Minimum Supply Temperature Setpoint
 
+This numeric field is the minimum value to which the supply setpoint can be reset.  This ensures that the hot water temperature does not go too low during reset operations.
+
+#### Maximum Supply Temperature Setpoint
+
+This numeric field plays multiple roles.  During initialization, such as before the plant data structures are fully established, this is used as the default plant supply setpoint temperature.  During normal calculation, this is used as the maximum hot water supply temperature to avoid resetting the setpoint too high.  During times of no-load, or odd conditions such as a negative load, the setpoint is reset to this design setpoint.
+
+#### Return Temperature Setpoint Input Type
+
+This alpha field must be one of three strings: "Constant", "Scheduled", or "ReturnTemperatureSetpoint".  If the choice is "Constant", the constant value in the next field is used as the target return temperature.  If the choice is "Scheduled", the current value of the schedule named in the following field is used as the target return temperature.  If the choice is "ReturnTemperatureSetpoint", the actual setpoint temperature on the "Plant Loop Supply Inlet Node" specified above is retrieved and used as the target return temperature.  This is convenient as a separate setpoint manager could be established to pre-calculate the desired return temperature, and will assign the Setpoint internally.  Then when this return temperature manager executes, the setpoint will be updated and utilized.
+
+#### Return Temperature Setpoint Constant Value
+
+If the "Design Hot Water Return Temperature Input Type" field is "constant", the numeric value entered here is used as the target return temperature throughout the simulation.
```

Comment:
Old field name is lingering here "Design Hot Water Return Temperature Input Type" --> Return Temperature Setpoint Input Type.  Happens twice here and twice in CHW object.